### ISSUE TITLE
Face Verification Model

### ISSUE THREAD

**Issue by binguliki** [abhisheks008/DL-Simplified#511]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Face Verification system using Siamese Neural Network </br>
:red_circle: **Aim** :  To develop a siamese neural network trained over triplet loss function for face verification task</br>
:red_circle: **Dataset** : [LFW dataset](https://vis-www.cs.umass.edu/lfw/) </br>
:red_circle: **Approach** :  Would be using LFW dataset to generate triplets for training the model and develop a Siamese Neural network architecture  which would be trained over a triplet loss function and then use this model for Facial Verification task 

Please Assign this issue to me as i'm currently working on this project and also do provide the GSSoC24 tag for my contribution to be counted.

- Full name :  Bingumalla Likith
- GitHub Profile Link : [binguliki](https://github.com/binguliki)
- Email ID : blikith86@gmail.com
- Participant ID (if applicable): NA
- Approach for this Project : GSSoC24
- What is your participant role? (Mention the Open Source program):
   GSSoC24 Contributor
